Tests/Unit Tests/JSON Validator/TWTJSONObjectValidatorTestCase.m

```diff
@@ -44,20 +44,30 @@ @implementation TWTJSONObjectValidatorTestCase
 - (void)testSuite
 {
     // TODO: make these relative
```

Comment:
I think we can accomplish it using a build setting and a preprocessor directive. Let’s look at this Monday.